internal/pkg/workspace/workspace.go

```diff
@@ -189,23 +189,9 @@ func (ws *Workspace) WritePipelineManifest(marshaler encoding.BinaryMarshaler) (
 	return ws.write(data, pipelineFileName)
 }
 
-// DeletePipelineManifest removes the pipeline manifest from the copilot/ directory.
-func (ws *Workspace) DeletePipelineManifest() error {
-	copilotPath, err := ws.copilotDirPath()
-	if err != nil {
-		return err
-	}
-
-	return ws.fsUtils.Remove(filepath.Join(copilotPath, pipelineFileName))
-}
-
-// DeleteService removes the service directory from the copilot/ directory.
-func (ws *Workspace) DeleteService(name string) error {
-	copilotPath, err := ws.copilotDirPath()
-	if err != nil {
-		return err
-	}
-	return ws.fsUtils.RemoveAll(filepath.Join(copilotPath, name))
+// DeleteWorkspaceFile removes the .workspace file under copilot/ directory.
+func (ws *Workspace) DeleteWorkspaceFile() error {
+	return ws.fsUtils.Remove(CopilotDirName + "/" + workspaceSummaryFileName)
```

Comment:
Out of curiosity, do we ever use `DeleteAll()` now that we are using this instead? If not, can we remove it? A quick check seems to suggest no but not sure if there are other use cases.

Reply:
Nope! I was thinking about not deleting it in case we would use it in the future but we can always rewrite the small method again. Will do!

Reply:
Can we use [filepath.Join](https://golang.org/pkg/path/filepath/#Join) when creating the path so that it works on the other platforms too:
```suggestion
	return ws.fsUtils.Remove(filepath.Join(CopilotDirName, workspaceSummaryFileName))
```